packages/daemon/test/lookup.js

```diff
@@ -0,0 +1,9 @@
+import { Far } from '@endo/far';
+
+export const make = () => {
+  return Far('Lookup', {
```

Comment:
Shirley, there’s a noun for that verb.

```suggestion
  return Far('Well', {
```

Reply:
Well, according to my sources it's also "lookup". And stop calling me Shirley!